src/bin/tikv-ctl.rs

```diff
@@ -0,0 +1,184 @@
+                                     .help("set the scan from raw key, in escaped format"))
+                                .arg(Arg::with_name("to")
+                                     .short("t")
+                                     .takes_value(true)
+                                     .help("set the scan end raw key, in escaped format"))
+                                .arg(Arg::with_name("limit")
+                                     .short("l")
+                                     .takes_value(true)
+                                     .help("set the scan limit"))
+                                .arg(Arg::with_name("cf")
+                                     .short("c")
+                                     .takes_value(true)
+                                     .help("column family name")))
+        .subcommand(SubCommand::with_name("print")
+                    .about("print the raw value")
+                    .arg(Arg::with_name("cf")
+                         .short("c")
+                         .takes_value(true)
+                         .help("column family name"))
+                    .arg(Arg::with_name("key")
+                         .short("k")
+                         .takes_value(true)
+                         .help("set the query raw key, in escaped form")))
+        .get_matches();
+
+    let db_path = matches.value_of("db").unwrap();
+    let db = util::rocksdb::open(db_path, ALL_CFS).unwrap();
+    if let Some(matches) = matches.subcommand_matches("print") {
+        let cf_name = matches.value_of("cf").unwrap_or("default");
+        let key = String::from(matches.value_of("key").unwrap());
+        dump_raw_value(db, cf_name, key);
+    } else if let Some(matches) = matches.subcommand_matches("raft") {
+        if let Some(matches) = matches.subcommand_matches("log"){
+            let region = String::from(matches.value_of("region").unwrap());
+            let index = String::from(matches.value_of("index").unwrap());
+            dump_raft_log_entry(db, region, index);
+        } else if let Some(matches) = matches.subcommand_matches("region") {
+            let region = String::from(matches.value_of("region").unwrap());
+            dump_region_info(db, region);
+        } else {
+            panic!("Currently only support raft log entry and scan.")
+        }
+    } else if let Some(matches) = matches.subcommand_matches("scan") {
+        let from = String::from(matches.value_of("from").unwrap());
+        let to = matches.value_of("to").map(|s| String::from(s));
+        let limit = matches.value_of("limit").map(|s| s.parse().unwrap());
+        let cf_name = matches.value_of("cf").unwrap_or("default");
+        if let Some(ref to) = to {
+            if to > &from {
+                panic!("The region's start pos must greater than the end pos.")
+            }
+        }
+        dump_range(db, from, to, limit, cf_name);
+    }
+    else {
+        panic!("The command is not supported now.");
```

Comment:
can we output the help message here?
I think we can return directly, no need panic.